logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
+logs. To utilize these features, create a `ClusterLogging` Custom Resource in
```

Comment:
s/utilize/use

I think this section should be separate procedure that contains the commands that you should run to create the resource instead of a hidden procedure in the assembly intro.

Reply:
Just thinking the same thing.

Reply:
First, I agree - this could be it's own subsection or book. I noticed, however, that we did the same thing in Enterprise: https://docs.openshift.com/container-platform/4.1/logging/config/efk-logging-configuring.html

Perhaps just move the relevant text to it's own assembly and a separate assembly for the `About` section? That's in line with how Enterprise is organized.